src/dss/manifest_templates/notebook_deployment.yaml.j2

```diff
@@ -23,6 +23,11 @@ spec:
         - env:
           - name: MLFLOW_TRACKING_URI
             value: {{ mlflow_tracking_uri }}
+          {% if gpu %}
+          resources:
+            limits:
+              {{ gpu }}: 1
+          {% endif %}
```

Comment:
Do we support only a single GPU per notebook?  If I have two GPUs, should we support using both?